### PR TITLE
Add a menu CTA block to the Programs demo menu item

### PR DESCRIPTION
**Pull request by aleevas**

### CODE REVIEW

File: config/install/migrate_plus.migration.y_lb_demo_events.yml [hunk 1]
```diff
@@ -19,9 +19,9 @@ source:
       summary: "All aboard, seniors! Take a 2-hour tour
         of the city on this casual riverboat picnic
         that includes a boxed lunch."
-      contact_email: sXyimJuV@example.com
-      contact_name: 'John Doe'
-      contact_phone: '+1 (888) 123-4567'
+      contact_email: events@ymcaspringfield.org
```
Comment (podarok) [inline config/install/migrate_plus.migration.y_lb_demo_events.yml:22]: There is no such domain
We need to keep example.org or register it in order not confuse end users or do not create false link if anyone would decide to play fraud and register domain to hijack traffic

File: config/install/migrate_plus.migration.y_lb_demo_events.yml [hunk 2]
```diff
@@ -34,9 +34,9 @@ source:
       title: 'Event Title'
       body: "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>"
       summary: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
-      contact_email: sXyimJuV@example.com
-      contact_name: 'John Doe'
-      contact_phone: '+1 (888) 123-4567'
+      contact_email: events@ymcaspringfield.org
```
Comment (podarok) [inline config/install/migrate_plus.migration.y_lb_demo_events.yml:37]: There is no such domain
We need to keep example.org or register it in order not confuse end users or do not create false link if anyone would decide to play fraud and register domain to hijack traffic

File: config/install/migrate_plus.migration.y_lb_demo_events.yml [hunk 3]
```diff
@@ -63,9 +63,9 @@ source:
...
```
Comment (podarok) [inline config/install/migrate_plus.migration.y_lb_demo_events.yml:66]: There is no such domain
We need to keep example.org or register it in order not confuse end users or do not create false link if anyone would decide to play fraud and register domain to hijack traffic

Comment (aleevas) [reply config/install/migrate_plus.migration.y_lb_demo_events.yml]: @podarok should be fixed in the last commit

File: config/install/migrate_plus.migration.y_lb_demo_events.yml [hunk 4]
```diff
@@ -48,9 +48,9 @@ source:
...
```
[the remark this answers is not in the document]
Comment (aleevas) [reply config/install/migrate_plus.migration.y_lb_demo_events.yml]: @podarok should be fixed in the last commit

File: config/install/migrate_plus.migration.y_lb_demo_events.yml (hunk 1 above)
Comment (aleevas) [reply config/install/migrate_plus.migration.y_lb_demo_events.yml]: @podarok should be fixed in the last commit

**Pull request merged**